Addition of a package generator function to the com_localise component

hikashop-nicolas opened the pull request:
The pull request adds new fields and a "download" button on the package edition interface so that you can fill the fields to be used in the language package and then then click on the "download" button in order to get a package generated on the fly and directly sent to your browser based on the settings of your package.

hikashop-nicolas committed:
Addition of a package generator function to the com_localise component

infograf768 commented:
This has to be updated to the state of the branch
Please explain this patch.
How to test it? What is one supposed to get and where?
Is it for core lang files or not?

infograf768 commented:
I managed to test this and it goes in the right direction  for core packs :)
We do have some issues though (for example the tpl ini files can't be included)
I suggest you update your branch as we can now see the default core.xml

hikashop-nicolas committed:
Merge branch 'develop' of https://github.com/joomla-projects/com_localise into develop

Conflicts:
	administrator/components/com_localise/views/package/view.html.php

hikashop-nicolas committed:
Adding saving of the language of the package
Removing the language button in the packages listing toolbar as it has never been implemented and no one seems to know why it's there

wilsonge commented:
also for the header() we should probably use `JFactory::getApplication()->setHeader()`
